src/index.js

```diff
@@ -1 +1,49 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const urlScore = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/uB09yxigYMjYQjf4BYmA/scores';
+const refresh = document.querySelector('.refresh');
+const submit = document.querySelector('.submit-btn');
+const list = document.querySelector('.list');
+
+function display(arr) {
+  list.innerHTML = '';
+  arr.forEach((element) => {
+    list.innerHTML += `
+      <li>${element.user} : ${element.score}</li>
+       `;
+  });
+}
```

Comment:
- [ ] According to ES6 syntax you are required to use arrow functions. Kindly refactor this function.